web-ui/test/unit/login/app.spec.js

```diff
@@ -0,0 +1,26 @@
+import { shallow } from 'enzyme';
+import expect from 'expect';
+import React from 'react';
+import { App } from 'src/login/app';
+
+describe('App', () => {
+  let app;
+  const mockTranslations = key => key;
+
+  beforeEach(() => {
+    app = shallow(<App t={mockTranslations} />);
+  });
+
+  it('renders login form', () => {
+    expect(app.find('form').props().action).toEqual('/login');
+  });
+
+  it('renders auth error message', () => {
+    app = shallow(<App t={mockTranslations} authError />);
```

Comment:
Should we keep the same name that we are using in the component (AppWrapper), just to keep consistency?